src/xai_grok/schemas/tool_function.py

```diff
@@ -0,0 +1,10 @@
+import typing
+
+import pydantic
+
+
+class ToolFunction(pydantic.BaseModel):
+    description: typing.Optional[str] = None
+    name: str
```

Comment:
The `parameters` field uses `typing.Dict[typing.Any, typing.Any]` which is too permissive. For a tool function schema, parameters should have a more specific type like JSON schema object or a structured type definition to ensure proper validation.